### ISSUE TITLE
Incompatibility with ftbchunks

### ISSUE THREAD

**Issue by SeseMueller** [spoorn/sodium-forge#32]
### Expected Behavior
The ftbChunks mod should be able to display the minimap 

### Actual Behavior
The minimap that ftbChunks shows doesn't update and draws chunks I visited in black, instead of actually showing a representation of what's in them

### Reproduction Steps
Go into a world with ftbchunks
-> the minimap doesn't work

### Attachments
![2021-06-19_13 29 57](https://user-images.githubusercontent.com/44344995/122641049-0c83bf80-d103-11eb-8b67-dfbc5a48d08d.png)

### [Optional] Server/Client logs
https://paste.ee/p/M2U6d

### Mods list
I have used the FTB presents Direwolf 20 1.16 modpack and added this sodium. 

### System Information
I have used this: https://gist.github.com/tanmayb123/d55b16c493326945385e815453de411a
way to get Minecraft running.

- Java Version: Zulu Java JDK for macOS ARM64 11.43.1015
- CPU: Apple M1
- GPU: Apple M1
- Sodium Reforged Version: sodium-foge-2.2

**Comment by skybix** [comment]
I seem to be having the same issue.

**Comment by spoorn** [comment]
The newer versions of FTB Chunks/Library (Alpha versions) seem to work with Sodium:
https://www.curseforge.com/minecraft/mc-mods/ftb-library-forge/files/all
https://www.curseforge.com/minecraft/mc-mods/ftb-chunks-forge/files/all

Unfortunately, according to the file comments it looks like upgrading to the Alpha version could break old worlds

**Comment by Darkmega18** [comment]
ouch. this is my issue too also. but I'll give those alphas a look then. since I make new worlds on the regular anyways for testing. :V

**Comment by SeseMueller** [comment]
I just tested it with the alpha versions of both mods. All versions from the "Mayor update" and up give the message "mod ftbchunks failed to load in correctly". The Error is: "Failure message: FTB Chunks (ftbchunks) has failed to load correctly
		java.lang.NoClassDefFoundError: dev/ftb/mods/ftbteams/property/TeamProperty". As this seems to be a problem with ftbteams, I looked at the forge versions of ftbteams. They don't have alpha versions and the last update was a while back. 
All versions from before the mayor update are releases and have the same issue as described in the original issue description.

@spoorn so sadly, there is no luck there. I didn't see them saying it was sodium compatible; do you remember where you read that?

**Comment by Darkmega18** [comment]
oh, really? I managed to download all alphas for ftb's mods and have it run fine with sodium. chunks, library, quest and teams? 
![image](https://user-images.githubusercontent.com/8104689/123139641-c5e2ec00-d499-11eb-994d-8fdca0e0ad5c.png)
those ones? I use forge 36.1.24 and like 170 other things. :V

**Comment by SeseMueller** [comment]
OK. So, I got it running. I overlooked the alpha versions of teams and assumed there were only releases. I'm sorry for that.

After I added the latest Alpha, I needed to update Ranks, but after that, it worked perfectly. 

In conclusion, at the current moment, alpha versions of **FTB Chunks, Library, Quests, Teams and Ranks** need to be used in order for FTBChunks to work properly with Sodium. The next releases should include the fixes that were made in the alpha versions. 

For that reason, I am closing this issue.
